app/src/main/java/clarkson/ee408/tictactoev4/MainActivity.java

```diff
@@ -26,20 +31,89 @@ public class MainActivity extends AppCompatActivity {
     private Button[][] buttons;
     private TextView status;
     private Gson gson;
+    private boolean shouldRequestMove;
+    private SocketClient socketClient;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         this.tttGame = new TicTacToe(STARTING_PLAYER_NUMBER);
         this.gson = new GsonBuilder().serializeNulls().create();
+        socketClient = SocketClient.getInstance();
+        shouldRequestMove = false;
 
         buildGuiByCode();
+        updateTurnStatus();
 
         Handler handler = new Handler();
         GameMoveTaskRunnable runnable = new GameMoveTaskRunnable(this, handler);
         handler.post(runnable);
     }
 
+    /**
++   * Sends a request to the server to ask for a game move made by the other player.
++   */
```

Comment:
Weird `+` is in the first column but shouldn't be

```suggestion
    /**
     * Sends a request to the server to ask for a game move made by the other player.
     */
```